backend/.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 .env
+.env.development
```

Comment:
Can we do 
```.gitignore
.env.* 
```

to stop all .env files from being tracked?

Reply:
Sure 👌🏻